templates/vars/aks/prod.json

```diff
@@ -18,7 +18,10 @@
       "value": "VirtualMachineScaleSets"
     },
     "vmSize": {
-      "value": "Standard_D4s_v3"
+      "value": "Standard_DS3_v2"
+    },
+    "osDiskType": {
+      "value": "Ephemeral"
```

Comment:
normally we wouldn't merge a PR like this till we had rolled it out to prod, i.e master reflects what's currently in production